.github/workflows/windows.yml

```diff
@@ -37,42 +40,6 @@ jobs:
     - name: checkout
       uses: actions/checkout@v2
 
-    # The Windows runners do not (yet) have a proper msys/mingw environment
-    # pre-configured like AppVeyor does, though they will likely be added in the future.
-    # https://github.com/actions/virtual-environments/issues/30
-    #
-    # In the interim, this ensures mingw32 is installed and available on the PATH
-    # for the i686-pc-windows-gnu target. This approach is used because it's common in
-    # other rust projects and there are issues/limitations with the msys2 chocolatey nuget
-    # package and numworks/setup-msys2 action.
-    # https://github.com/rust-lang/rust/blob/master/src/ci/scripts/install-mingw.sh#L59
-    # https://github.com/rust-lang/rustup/blob/master/appveyor.yml
-    #
-    # Use GitHub Actions cache support to avoid downloading the .7z file every time
-    # to be cognizant of the AWS egress cost impacts
-    # https://help.github.com/en/actions/automating-your-workflow-with-github-actions/caching-dependencies-to-speed-up-workflows#usage-limits-and-eviction-policy
```

Comment:
MSYS2 has subsequently been added to the Actions runners so all the mingw tooling is available for the `i686-pc-windows-gnu` target